about.html

```diff
@@ -0,0 +1,123 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <script src="https://kit.fontawesome.com/d31edfa6bf.js" crossorigin="anonymous"></script>
+  <script src="https://cdn.jsdelivr.net/npm/bootstrap@5.0.2/dist/js/bootstrap.bundle.min.js" integrity="sha384-MrcW6ZMFYlzcLA8Nl+NtUVF0sA7MsXsP1UyJoMp4YLEuNSfAP+JcXn/tWtIaxVXM" crossorigin="anonymous"></script>
+  <link rel="stylesheet" href="path/to/font-awesome/css/font-awesome.min.css">
+  <link href="https://fonts.googleapis.com/css?family=Lato" rel="stylesheet">
+  <link rel="stylesheet" href="CSS/style.css">
+  <link rel="stylesheet" href="CSS/About.css">
+  <link rel="stylesheet" href="CSS/media.css">
+  <title>MMF 2022 | Mawazine Music Festival</title>
+  <link rel="icon" type="image/png" href ="/Assets/mawazine.png">
+</head>
+<body>
+  <header>
+    <div class="menu">
+      <div></div>
+    </div>
+    <nav class="first">
+      <ul>
+        <li><a href=""><i class="fa fa-twitter fa-x"></i></a></li>
+        <li><a href=""><i class="fa fa-facebook fa-x"></i></a></li>
+        <li> <select name="languages" id="lang">
+          <option value="English" selected>English</option>
+          <option value="Arabic">Arabic</option>
+          <option value="Spanish">Spanish</option>
+          <option value="French">French</option>
+        </select></li>
+        <li>My page</li>
+        <li>logout</li>
+      </ul>
+    </nav>
+    <div class="second">
+      <img class="logo" src="Assets/mawazine.png" alt="logo">
+      <nav class="nav-bar">
+        <ul>
+          <li><a href="index.html" class="nav-link" id="Home">Home</a></li>
+          <li><a href="#" class="nav-link">About</a></li>
+          <li><a href="#Program" class="nav-link">Program</a></li>
+          <li><a href="#Geusts" class="nav-link" >Guests</a></li>
+          <li><a class="nav-link" href="#Parteners">Partners</a></li>
+          <li><a href="#Envents" class="nav-link">Event</a></li>
+          <li><a href="#" class="nav-link"><span id="mawazine">mawazine</span> </a></li>
+        </ul>
+      </nav>
+    </div>
+  </header>
+  <main>
```

Comment:
Consider making the main section as per the original design. Try to be consistent with the colors used in the original design.
| Your design  | Original design |
| ------------- | ------------- |
| ![image](https://user-images.githubusercontent.com/33268927/182564372-a4747015-9590-4250-97c2-21e56be4dd2f.png) | ![image](https://user-images.githubusercontent.com/33268927/182564197-7cfbd1dc-34eb-4f50-8bc6-7d1554f152b7.png) |